src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj

```diff
@@ -411,23 +411,23 @@
     <Compile Update="PropPages\AdvancedServicesDialog.vb">
       <SubType>UserControl</SubType>
     </Compile>
-    <Compile Update="proppages\AdvBuildSettingsPropPage.vb">
+    <Compile Update="PropPages\AdvBuildSettingsPropPage.vb">
```

Comment:
Do we even need these files to be listed any more?

Reply:
Possibly not. There's a lot of metadata on these items.